Makefile

```diff
@@ -24,20 +24,28 @@ deps:
 checks:
 	./scripts/checks "$(PKGS)"
 
-build: deps checks
+run-build: deps checks
 	godep go install -v $(TO_BUILD)
 
+build: start
+	vagrant ssh netplugin-node1 -c 'sudo -i bash -lc "source /etc/profile.d/envvar.sh && cd /opt/golang/src/github.com/contiv/netplugin && make run-build"'
```

Comment:
> I don't see how this is any different from typical vagrant usage

Yes, this is not different in behavior. The only difference is wrt user's experience. In case of doing `vagrant up` and then deleting the workspace we can consider it to be error on user's part if it leaves behind a vm. But if a user does `make build` and then delete their workspace which leaves behind a vm, it is not an error on their part.

You are right there is no easy way to achieve both in this case i.e. protect the user or improved build time. We can may be print a informative message about the running vm after build to alleviate this a bit. For now, I am ok starting with what we have here as it does solves the builds on mac.

ping @jainvipin @shaleman as well to get their thoughts.